src/realm/util/ez_websocket.hpp

```diff
@@ -21,17 +21,19 @@ struct EZConfig {
     std::mt19937_64& random;
     util::network::Service& service;
     std::string user_agent;
-    bool tcp_no_delay;
 };
 
 struct EZEndpoint {
     std::string address;
     port_type port;
-    std::string http_host; // Contents of `Host:` request header
-    std::string path;
+    std::string path;      // Includes auth token in query.
```

Comment:
I propose we test this change if possible

Reply:
This is just code motion. The Host header was being computed in the sync layer. I took the same logic and just moved it down to the websocket implementation layer. There shouldn't be any change to how the Host header is computed, so existing coverage should be sufficient.